soc/xtensa/intel_adsp/cavs_v15/Kconfig.defconfig.series

```diff
@@ -65,6 +65,12 @@ endif # LOG
 
 if SMP
 
+config SMP_BOOT_DELAY
+	default SOF
+
+config CORE_COUNT
```

Comment:
this is using an SOF Kconfig that is duplicated below, we need to fix that. Maybe in SOF itself set CORE_COUNT to what Zephyr provide using MP_NUM_CPUS

Reply:
@lyakh @marc-hb please align fully with Zephyr on all Kconfig symbols - i.e. SOF will follow the Zephyr naming/usage.